client/src/pages/SubmissionsPage.vue

```diff
@@ -121,6 +122,16 @@
                         "
                         >{{ $t("submission.action.accept_for_review") }}</q-item
                       >
+                      <q-item
+                        v-if="submission.status != 'INITIALLY_SUBMITTED'"
```

Comment:
I'm noticing that this "Accept as Final" button as well as the other buttons still appear for a submission whose review has not yet been opened, which is inconsistent with the Submission Review page's display of the "Open for Review" button only. I think the available options should be consistent between the Submission Review page and the Submissions page.

<img width="266" alt="A user on the Submissions page can choose options meant only for under review when a submission is out of review" src="https://user-images.githubusercontent.com/2668987/184443250-d149dcbe-b2e3-466e-a9c3-9bcae2478441.png">

Reply:
I think that makes sense to keep the options consistent. Made the suggested changes.